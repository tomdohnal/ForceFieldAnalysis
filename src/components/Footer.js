import { createComponent } from 'react-fela';

const footer = () => ({
  position: 'fixed',
  height: '130px',
  width: '100%',
  background: '#268bd2',
  bottom: 0,
  padding: '40px 16px',
});

export default createComponent(footer);
