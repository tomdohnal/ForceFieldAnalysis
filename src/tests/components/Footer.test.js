import React from 'react';
import { shallow } from 'enzyme';

import Footer from '../../components/Footer';

describe('Footer', () => {
  it('renders correctly', () => {
    const tree = shallow(<Footer drivingForcesCount={5} hinderingForcesCount={5} centerText="It's a tie!" />);

    expect(tree).toMatchSnapshot();
  });
});
