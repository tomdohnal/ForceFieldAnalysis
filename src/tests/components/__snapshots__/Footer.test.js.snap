// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footerStyles>
  <Grid
    columns={3}
  >
    <GridColumn>
      <text
        color="#FFF"
        fontSize="54px"
      >
        Driving forces: 
        5
      </text>
    </GridColumn>
    <GridColumn
      textAlign="center"
    >
      <text
        color="#FFF"
        fontSize="24px"
      >
        It's a tie!
      </text>
    </GridColumn>
    <GridColumn
      textAlign="right"
    >
      <text
        color="#FFF"
        fontSize="54px"
      >
        Hindering forces: 
        5
      </text>
    </GridColumn>
  </Grid>
</footerStyles>
`;
